pymc4/inference/sampling/sample.py

```diff
@@ -10,7 +10,8 @@ def sample(model,
            num_leapfrog_steps=3,
            numpy=True):
     initial_state = []
-    for name, shape in model.unobserved.iteritems():
+    for name in model.unobserved.keys():
```

Comment:
use `for name, (_, shape, _) in model.items():` to indicate that `dist` and `rv` are not used in the loop